C/services/south/ingest.cpp

```diff
@@ -720,12 +721,30 @@ void Ingest::processQueue()
 					// Block of code to execute holding the mutex
 					lock_guard<mutex> guard(m_qMutex);
 					std::vector<Reading *> *newQ = new vector<Reading *>;
+					// Recover buffer memory generated to ingest data
```

Comment:
This breaks the entire point of the change that added the issue, sending the data in the shutdown.